OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,22 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class SkirmishSettings
```

Comment:
I only assume it could be useful, but it could be removed. E.g. I play MP as one faction, but I try/practice another in Skirmish.